extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -668,29 +668,48 @@ export class SubscriptionService {
   }
 
   /**
-   * Performs pingback on local platform.
+   * Performs pingback on all platform.
    * @return {?Promise}
    * @private
    */
   performPingback_() {
     if (this.viewTrackerPromise_) {
-      const localPlatform = this.platformStore_.getLocalPlatform();
+      const availablePlatforms = this.platformStore_.getAvailablePlatforms();
```

Comment:
Let's clarify local vs subscribe.google.com placement first. If local - this code would be different (actually probably more similar to what it was before).